perf/benchmark/templates/fortio.yaml

```diff
@@ -73,6 +73,11 @@ metadata:
     app: {{ $.name }}
   name: {{ $.name }}
 spec:
+{{- if eq $.name "fortioclient" }}
+replicas: 4
```

Comment:
why hardcoded to 4 and 20 instead of just `$.replicas` or something?

Reply:
just a wip pr, updated!